sdk/core/core-tracing/karma.conf.js

```diff
@@ -0,0 +1,116 @@
+// https://github.com/karma-runner/karma-chrome-launcher
+process.env.CHROME_BIN = require("puppeteer").executablePath();
+
+module.exports = function(config) {
```

Comment:
I didn't bother too much, or at all, with `karma.conf.js`. I just took what is in core-rest-client as is.